README.md

```diff
@@ -159,6 +159,7 @@ You can see in which language an app is written. Currently there are following l
 - [Postgres.app](https://github.com/PostgresApp/PostgresApp) - The easiest way to get started with PostgreSQL on the Mac. ![SwiftIcon]
 - [Redis.app](https://github.com/jpadilla/redisapp) - The easiest way to get started with Redis on the Mac. ![SwiftIcon]
 - [Sequel Pro](https://github.com/sequelpro/sequelpro) - MySQL/MariaDB database management for macOS.  ![ObjectiveCIcon]
+- [DB Browser for SQLite](https://sqlitebrowser.org) - SQLite database management GUI. ![CppIcon]
```

Comment:
Can you change the link to GitHub repository?

Reply:
Done.  We may be moving off of GitHub now that it's been bought by MS, but we'll cross that bridge when we come to it. :smile:

Reply:
Agree 👍

Reply:
@laughedelic Well, we tend to avoid relying on groups that have a bad track record.  Microsoft's is pretty terrible.  They're not at Oracle level, but not far off it.

Reply:
Hi @justinclift. Thanks for clarifying your position. I didn't really want to initiate a discussion, just left a reaction. My opinion about this is that judging companies as a whole by some kind of one-dimensional summary of their record doesn't make much sense. I'm not a fan of MS, but I think it's not fair to just say that they have a "pretty terrible" track record. 
In the end it's more about people. There are a lot of good people at MS that have done a lot of amazing work (including some great research). I recommend you to read this thread where the future Github CEO (from MS) answers questions: https://www.reddit.com/r/AMA/comments/8pc8mf/im_nat_friedman_future_ceo_of_github_ama/

P.S. I'm in no way convincing you not to move off of Github, there are some good alternatives, just saying that _IMO_ acquisition by Microsoft is not a very good reason on its own.

Reply:
Oh, I get all that.  There's just no way that a company which right **to this very day** is still engaged in terrible practises, and **will not stop** regardless of backlash.  eg forced upgrading of people to Win 10, which now steals data and **can't be turned off**.  There is literally no excuse for that.

Reply:
Just to clarify, when a company does crap like that, it definitely writes off any positive actions by small groups doing their best.  Just my point of view. :smile: